### PR TITLE
Typos in doc /doc/dev/tests.md

### LINKED ISSUE #32489: Typos in doc /doc/dev/tests.md
First typo:

- This page: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/tests.md#tox
- shows: (env) > pip install tox<5
- What is the "<5"? I think it's a typo and should be removed.

Second typo:

- This page: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/tests.md#environment-prerequisites
- Shows: "Git version > 2.30.0 is to on the machine and in the path"
- I think it was meant to say "installed on the machine" instead of "is to on the machine"

### PR DESCRIPTION
**Pull request by shubhusion**
Fixes #32489

# Description

Removed Typos and linting errors in doc /doc/dev/tests.md

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by shubhusion** [comment]
@microsoft-github-policy-service agree

**Comment by shubhusion** [comment]
@microsoft-github-policy-service agree

**Review by dargilco — APPROVED** [review]

File: doc/dev/tests.md [hunk 1]
```diff
@@ -95,58 +97,65 @@ We should now be able to open an interactive Python terminal, and execute code f
...
+
 ```cmd
 (env) azure-sdk-for-python\sdk\my-service\my-package> pytest <test_file.py> -s
 ```
 
 ## Tox
 
 The Python SDK uses the [tox project](https://tox.wiki/en/latest/) to automate releases, run tests, run linters, and build our documentation. The `tox.ini` file is located at `azure-sdk-for-python/eng/tox/tox.ini` for reference. You do not need to make any changes to the tox file for tox to work with your project. Tox will create a directory (`.tox`) in the head of your branch. The first time you run tox commands it may take several moments, but subsequent runs will be quicker. To install tox run the following command from within your virtual environment.
-`(env) > pip install tox<5`.
+`(env) > pip install tox`.
```
Comment (kristapratico) [inline doc/dev/tests.md:132]: I think we just need to wrap tox<5 in quotes: `pin install "tox<5"`

File: doc/dev/tests.md [hunk 2]
```diff
@@ -196,19 +206,19 @@ be found in the [package's README][python-dotenv_readme].
...
+   section of the "Subscriptions" blade in the [Azure Portal][azure_portal].
 2. Define the `AZURE_TENANT_ID`, `AZURE_CLIENT_ID`, and `AZURE_CLIENT_SECRET` of a test service principal. If you do not
-have a service principal, use the Azure CLI's [az ad sp create-for-rbac][azure_cli_service_principal] command (ideally,
-using your alias as the service principal's name prefix):
-```
-az login
+   have a service principal, use the Azure CLI's [az ad sp create-for-rbac][azure_cli_service_principal] command (ideally,
+   using your alias as the service principal's name prefix):
+
+```az login
```
Comment (kristapratico) [inline doc/dev/tests.md:221]: This change causes the `az login` to be removed from the example:

image

File: doc/dev/tests.md [hunk 3]
```diff
@@ -346,7 +356,7 @@ not locally available.
 ##### Environment prerequisites
 
 - The targeted library is already migrated to use the test proxy.
-- Git version > 2.30.0 is to on the machine and in the path. Git is used by the script and test proxy.
+- Git version > 2.30.0 installed on the machine and in the path. Git is used by the script and test proxy.
```
Comment (mccoyp) [inline doc/dev/tests.md:359]: Thanks for the catch! This was originally supposed to be "is on", but "is installed on" is a fine fix
```suggestion
- Git version > 2.30.0 is installed on the machine and in the path. Git is used by the script and test proxy.
```

**Review by mccoyp — COMMENTED** [review]
+1 to Krista's comments

**Review by kristapratico — APPROVED** [review]

**Review by mccoyp — APPROVED** [review]

**Comment by kristapratico** [comment]
Thanks for your contribution @shubhusion! All looks good so I will merge. :)

**Pull request merged**